lib/puppeteer_environment.js

```diff
@@ -0,0 +1,33 @@
+const chalk = require('chalk')
```

Comment:
Nit: Can these files live in `/lib/puppeteer/*.*` ?

Reply:
done